emote/sac.py

```diff
@@ -126,6 +126,7 @@ def __init__(
         )
 
     def begin_batch(self, next_observation, rewards, masks):
+        # print(next_observation)
```

Comment:
This looks like a forgotten debug statement. Remove?

Reply:
fixed